sql/database.sql

```diff
@@ -10610,13 +10611,17 @@ CREATE TABLE `login_mfa_registrations` (
 DROP TABLE IF EXISTS `api_token`;
 CREATE TABLE `api_token` (
     `id`           bigint(20) NOT NULL AUTO_INCREMENT,
-    `token_api`    varchar(40),
+    `token_api`    varchar(4),
     `user_id`      bigint(20) NOT NULL,
     `patient_id`   bigint(20) NOT NULL,
     `token`        varchar(256) DEFAULT NULL,
     `token_auth`   varchar(255),
     `expiry`       datetime NULL,
-    PRIMARY KEY (`id`)
+    PRIMARY KEY (`id`),
+    UNIQUE KEY `token` (`token`),
+    KEY `token_api` (`token_api`),
+    KEY `user_id` (`user_id`),
+    KEY `patient_id` (`patient_id`)
```

Comment:
@sjpadgett ,
Was hoping to get your thoughts on these keys. The table will essentially explode in number of rows when users use the api, so have indexed the fields that will continue to ensure good performance on the queries of this table and the above patient_access_onsite table. Regarding the patient_access_onsite table, do you see any issues with making that pid key unique?

Reply:
Actually, upon further thought, I only need just one index here, which is:
```
UNIQUE KEY `token` (`token`),
```
That one index in combination with the above `UNIQUE KEY `pid` (`pid`)` in the patient_access_onsite should keep the queries in AuthRestController.php super fast.